karapace/protobuf/known_dependency.py

```diff
@@ -0,0 +1,163 @@
+"""
+karapace - known_dependency
+
+Copyright (c) 2023 Aiven Ltd
+See LICENSE for details
+"""
+
+# Support of known dependencies
+
+from enum import Enum
+from typing import Any, Dict, Set
+
+
+def static_init(cls: Any) -> object:
+    if getattr(cls, "static_init", None):
+        cls.static_init()
+    return cls
+
+
+class KnownDependencyLocation(Enum):
```

Comment:
No, but this info may be used as a reference in case we need to regenerate KnownDependency class which is generated by google protobuf predefined(known) types.